### PR TITLE
chore: Updated another set of third party urls

### PR DESCRIPTION
**Pull request by sagar-qa007**
## Description
Updated third party URLs.

Fixes `[Issue URL](https://github.com/appsmithorg/appsmith/issues/36437)`

## Automation

/ok-to-test tags="@tag.All"

### :mag: Cypress test results

> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/11174864458>
> Commit: c6432d4363f80513aae2d55310e5905dc83d0acc
> <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=11174864458&attempt=2" target="_blank">Cypress dashboard</a>.
> Tags: `@tag.All`
> Spec:
> <hr>Fri, 04 Oct 2024 07:58:04 UTC

## Communication
Should the DevRel and Marketing teams inform users about this change?
- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **Bug Fixes**
	- Enhanced fetch tests to ensure proper handling of requests and responses.
	- Corrected expected outputs for URL and date formats in the Table Widget tests.
	- Simplified validation logic for unsupported file types in the Document Viewer tests.

- **Chores**
	- Modified test cases to utilize local mock APIs for enhanced testing reliability.
	- Updated image URLs in fixture data to reflect new local sources.
	- Adjusted default images for widgets to ensure consistent visual representation.
	- Improved interactivity in modal widget tests with updated actions and dynamic data binding.
	- Updated limited tests to focus on server-side JavaScript function execution.

### CODE REVIEW

**Comment by sagar-qa007** [comment]
> Workflow run: https://github.com/appsmithorg/appsmith/actions/runs/11119069401. Cypress dashboard url: [Click here!](https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=11119069401&attempt=1) All Cypress tests have passed 🎉🎉🎉
> 
> ```
> ***** Repeat Run Summary ***** Total Tests with repeat: 36 Total Passed: 24 Total Failed: 12 Total Skipped: 0 *****************************
> ```

Please ignore this failure as changes reverted for this file

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts [hunk 1]
```diff
@@ -16,10 +16,9 @@ import EditorNavigation, {
 
 const ppt =
   "https://ssz.sgp1.digitaloceanspaces.com/3ZEO2582C29EA0KKK2/ppt-on-population-pptxafa26c44-208f-46a3-89cc-8a5c020b6863.pptx";
-const pngImage =
-  "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRb8umIiCU_K6ac-xS-ni4y6SP7XAd8C7Ms3Q&usqp=CAU";
+const pngImage = "https://www.appsmith.com/favicon.png";
 const jpgImage =
-  "https://community.appsmith.com/sites/default/files/styles/small_thumbnail/public/2024-03/aws-logo.jpg?itok=yG4bpfFs";
+  "https://community.appsmith.com/sites/default/files/styles/small_thumbnail/public/2024-03/aws-logo.jpg";
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts:21]: these third party urls are okay? @sagar-qa007

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV1/TableBugs_Spec.ts [hunk 2]
```diff
@@ -30,32 +30,35 @@ describe(
...
       });
 
       _.table.AssertURLColumnNavigation(
         0,
         0,
-        "https://wallpaperaccess.com/full/1376499.jpg",
+        "https://docs.appsmith.com/img/cube-logo_S50__hLNq.jpeg",
       );
       // _.table.AssertURLColumnNavigation(
       //   3,
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV1/TableBugs_Spec.ts:59]: remove commented code

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts (hunk 1 above)
Comment (sagar-qa007) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts:21]: These are appmsith urls. We can keep this as it is in our control.

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV1/TableBugs_Spec.ts (hunk 2 above)
Comment (sagar-qa007) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV1/TableBugs_Spec.ts]: @ApekshaBhosale Changes are done.

**Review by ApekshaBhosale — APPROVED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Modal/Modal_spec.ts [hunk 3]
```diff
@@ -18,9 +18,9 @@ describe(
   { tags: ["@tag.Widget", "@tag.Modal"] },
   function () {
     const image = (src: string) => 'img[src="' + src + '"]';
-    const jpgImg = "https://jpeg.org/images/jpegsystems-home.jpg";
-    const gifImg =
-      "https://mir-s3-cdn-cf.behance.net/project_modules/max_1200/5eeea355389655.59822ff824b72.gif";
+    const jpgImg =
```
Comment (mohanarpit) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/Modal/Modal_spec.ts:21]: @sagar-qa007 QQ: These images are still external to the Appsmith product. What would happen if the Devrel team decided to change or remove this image? Wouldn't this still lead to flakiness in the product? 

Would it be possible to bring these images in control of TED? Or is there a technical limitation in doing so?

Comment (sagar-qa007) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Modal/Modal_spec.ts:21]: We initially attempted to store the Docker images locally, but the test case requires loading an image via HTTPS for the deployed application. While adding the images to a CDN was considered, it was not deemed an ideal solution. However, the logos selected are from stable sources that are unlikely to change frequently, minimizing potential issues with third-party hosting within appsmith.

Comment (mohanarpit) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Modal/Modal_spec.ts:21]: @sagar-qa007 Why was a CDN solution deemed not ideal? Can you throw some light into this?

**Pull request merged**